.github/workflows/build-nightly-images.yaml

```
@@ -0,0 +1,141 @@
+
+      - name: Download provider image artifacts
+        uses: actions/download-artifact@v4
+        with:
+          pattern: ${{ steps.get_tag.outputs.download_pattern }}
+          path: /tmp/images
+          merge-multiple: true
+      
+      - name: Build manifest list and save
+        shell: bash
+        id: build_image
+        run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2027:warning:18:116: The surrounding quotes actually unquote this. Remove or escape them

(shellcheck)